react/components/SavedCard.tsx

```diff
@@ -25,10 +32,13 @@ const PaymentSystemIcon: React.FC = () => (
   </svg>
 )
 const SavedCard: React.FC<Props> = ({ lastDigits }) => {
+  const intl = useIntl()
   return (
     <div className="flex">
       <PaymentSystemIcon />
-      <div className="ml5">Cartão de Crédito final {lastDigits} </div>
+      <div className="ml5">
+        {`${intl.formatMessage(messages.creditCardLabel)} ****${lastDigits}`}
```

Comment:
no need for string interpolation
```suggestion
        {intl.formatMessage(messages.creditCardLabel)} ****{lastDigits}
```